src/main/webapp/i18n/en/global.json

```diff
@@ -90,6 +90,16 @@
         },
         "ribbon": {
             "dev":"Development"
+        },
+        "attribute" : {
+            "title" : "Attributes",
+            "key" : "Attribute Key",
+            "value" : "Attribute Value",
+            "add" : "Add",
+            "error" : {
+                "alreadyExist" : "This attribute key already exist in the list"
+            }
+
```

Comment:
Can placeholders be added to the `nl` version of this and the other JSONs for the added properties? This way we keep them in sync.